internal/DateTime/tests/DateTimeDecorator-specs.js

```diff
@@ -5,9 +5,9 @@ import {DateTimeDecorator} from '../';
 
 describe('DateTimeDecorator', () => {
 	test('should accept an updated JavaScript Date for its value prop', () => {
-		const Picker = DateTimeDecorator({}, function PickerBase ({locale, title, value}) {
+		const Picker = DateTimeDecorator({}, function PickerBase ({title, value}) {
```

Comment:
Why is this change needed?

Reply:
This change is added to avoid travis failure. As https://github.com/enactjs/sandstone/commit/bb662bc16a61b98746405425ed21c481a6addf82 only applied to develop branch.